web/client/components/data/identify/IdentifyContainer.jsx

```diff
@@ -58,11 +58,13 @@ module.exports = props => {
         showCoordinateEditor,
         onChangeClickPoint,
         onChangeFormat,
-        formatCoord
+        formatCoord,
+        showInMapPopup,
+        isCesium
     } = props;
 
     const latlng = point && point.latlng || null;
-
+    const isEnabled = enabled && (isCesium || !showInMapPopup);
```

Comment:
Could we move the `enabled && (isCesium || !showInMapPopup)` to the plugin selector? So the component is not awere of cesium or showInMapPopup but it will recieve only an `enable` prop